cf/data/array/h5netcdfarray.py

```diff
@@ -0,0 +1,85 @@
+# REVIEW: h5: `H5netcdfArray`: New class to access netCDF with `h5netcdf`
+import cfdm
+
+from ...mixin_container import Container
+from .locks import netcdf_lock
+from .mixin import ActiveStorageMixin, ArrayMixin, FileArrayMixin, IndexMixin
+
+
+class H5netcdfArray(
+    ActiveStorageMixin,
+    IndexMixin,
+    FileArrayMixin,
+    ArrayMixin,
+    Container,
+    cfdm.H5netcdfArray,
+):
+    """A netCDF array accessed with `h5netcdf`.
+
+    **Active storage reductions**
+
+    An active storage reductionx may be enabled with the `actify`
+    method. See `cf.data.collapse.Collapse` for details.
+
+    .. versionadded:: NEXTVERSION
+
+    """
+
+    def __dask_tokenize__(self):
+        """Return a value fully representative of the object.
+
+        .. versionadded:: NEXTVERSION
+
+        """
+        return super().__dask_tokenize__() + (self.get_mask(),)
```

Comment:
Hi Sadie, You're right :). The thing here is that `super().__dask_tokenize__()` is already a `tuple`, so using the `+` operator extend that tuple, rather than creating a more complicated tuple structure.

Reply:
Aha, to be honest I didn't realise you could concatenate tuples together, assuming I guess that because they are immutable it wouldn't be allowed, and assumed we were working with some other object due to that. So I learnt something new! Thanks for clarifying. Marking as resolved.